<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automotoras Diuca</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionModulo_Clientes"/>
    <addaction name="actionMarcas"/>
    <addaction name="actionModelosAutos"/>
    <addaction name="separator"/>
    <addaction name="actionLogin"/>
   </widget>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionModulo_Clientes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modulo Clientes</string>
   </property>
  </action>
  <action name="actionMarcas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modulo Marcas de Autos</string>
   </property>
  </action>
  <action name="actionModelosAutos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modulo Modelos de Autos</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Iniciar Sesion</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
